tech-far-hub/src/pages/index.tsx

```diff
@@ -1,15 +1,136 @@
+          {node.frontmatter.description}
+        </Initiative>
+      );
+    }
+  });
   return (
     <Layout>
-      <h1>Hello world</h1>
+      <Grid row gap={6} className="tfh-hp-top">
+        <Grid tablet={{ col: 8 }} className="tfh-hp-highlight">
+          <h2 className="tfh-hp-highlight-h2 font-heading-xl">
+            <Link to="/resouces/case-studies/va-user-research">
+              Case Study: VA.Gov Modernization Comparative Analysis
+            </Link>
+          </h2>
+          <p className="tfh-hp-highlight-desc">
+            The VA technology Acquisition Cent, in support of VA OIT/DSVA, utilized FAR 13.1 Simplified Acquisition
+            Procedures, which enabled the team to streamline a multi-step evaluation with onsite demonstrations, and
+            documented the evaluation using a comparative analysis.
+          </p>
+          <StaticImage
+            src="../../static/assets/img/ux-indonesia-8mikJ83LmSQ-unsplash.jpg"
+            width={623}
+            alt="Two people collaborating over graphs"
+          />
+        </Grid>
+        <Grid tablet={{ col: 4 }} className="tfh-hp-right-nav">
+          <hr className="text-accent-warm bg-accent-warm" />
+          <h3>01. Get Started</h3>
+          <ol>
+            <li>
+              <a href="#">What is the TechFAR Hub?</a>
+            </li>
+            <li>
+              <a href="#">How to Use the TFH</a>
+            </li>
+            <li>
+              <a href="#">How to Get Involved</a>
+            </li>
+          </ol>
+          <hr className="text-accent-cool-dark bg-accent-cool-dark" />
+          <h3>02. TFH Lifecycle</h3>
+          <p>See how the TechFAR Hub takes an agile approach to digital acquisition.</p>
+          <ol>
+            <li>
+              <Link to="/pre-solicitation/">Pre-Solicitation</Link>
+            </li>
+            <li>
+              <Link to="/solicitation/">Solicitation</Link>
+            </li>
+            <li>
+              <Link to="/evaluation/">Evaluation</Link>
+            </li>
+            <li>
+              <Link to="/contract-administration/">Contract Administration</Link>
+            </li>
+          </ol>
+          <hr className="text-green bg-green " />
+          <h3>03. Resources</h3>
+          <p>View first-hand experiences of fellows acquisition professionals, get tools, access training, and more</p>
+          <ol>
+            <li>
+              <Link to="/resources/tools/agile-estimator">Agile Estimator</Link>
+            </li>
+            <li>
+              <Link to="/resources/tools/sources-sought">Sources Sought Tool</Link>
+            </li>
+          </ol>
+        </Grid>
+      </Grid>
+      <Grid row>
+        <Grid col="fill" className="tfh-hp-initiatives">
+          <h2>Initiatives</h2>
+          {initiatives}
+        </Grid>
+      </Grid>
     </Layout>
   );
 };
 
 export default IndexPage;
 
 export const Head: HeadFC = () => <title>TechFAR Hub</title>;
```

Comment:
I'm still getting used to the new Head API

Reply:
I clearly am not really using it yet, but I need to sooooon